### DISCUSSION TITLE
Building a smart home control centre around Home Assistant | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#191] [Q&A]
# Building a smart home control centre around Home Assistant | Seeed Studio Wiki

Building a smart home control centre around Home Assistant

https://wiki.seeedstudio.com/h68k-ha-esphome/

**Comment by Shurov** [comment]
Please add ESPhome support for the [Grove Water Level](https://wiki.seeedstudio.com/Grove-Water-Level-Sensor/) (10cm) sensor

**Reply by MatthewJeffson** [reply]
We will consider that. Thank you for your interests in our product!
